articles/cms/wordpress/configuration.md

```diff
@@ -46,7 +59,7 @@ You must first create an app in the Auth0 dashboard before you can configure the
 
 ### Connections
 
-Enable the supported social identity providers you want to allow users to login with. You can configure your own app keys and settings for these connections in the [Auth0 Dashboard](${manage_url}/#/connections/social).
+Enable the supported social identity providers you want to allow users to login with. You can configure your own app keys and settings for these connections in the [Auth0 Dashboard](http://${manage_url}/#/connections/social).
```

Comment:
@khorne3 Same thing here, the default value of `${manage_url}` should be `https://manage.auth0.com` so the `http://` is not needed. The URL at the live doc navigated at `https://manage.auth0.com/#/connections/social`.